src/rollback_to_stable/rts_btree_walk.c

```diff
@@ -121,18 +121,27 @@ __rts_btree_walk_page_skip(
 static int
 __rts_btree_walk(WT_SESSION_IMPL *session, wt_timestamp_t rollback_timestamp)
 {
+    struct timespec rollback_timer;
     WT_DECL_RET;
     WT_REF *ref;
+    uint64_t rollback_msg_count;
+
+    /* Initialize the verbose tracking timer. */
+    __wt_epoch(session, &rollback_timer);
+    rollback_msg_count = 0;
 
     /* Walk the tree, marking commits aborted where appropriate. */
     ref = NULL;
     while (
       (ret = __wt_tree_walk_custom_skip(session, &ref, __rts_btree_walk_page_skip,
          &rollback_timestamp, WT_READ_NO_EVICT | WT_READ_VISIBLE_ALL | WT_READ_WONT_NEED)) == 0 &&
-      ref != NULL)
+      ref != NULL) {
+        /* Log a rts walk progress message. */
```

Comment:
This comment is redundant

Reply:
removed [d5dd416](https://github.com/wiredtiger/wiredtiger/pull/9194/commits/d5dd416373c4265ab17ca5b8fe0a83ebf40689cd)